app/retail/templates/emails/bounty_roundup.html

```diff
@@ -108,7 +108,7 @@
         </small>
       </div>
       </div>
-    {% endif %}
+    {% endif %}card card-user shadow-sm border-0
```

Comment:
Seems you have a few classes playing outside  in plain text :smiley:

Reply:
Also, I believe this PR is meant to solve
> enhancement to the pre-hack dev experience

with user story: 
> As a user I 'dl ike to know who are the notable participants so that I can decide whether to participate in the hackathon.

However, if I do not register for a hackathon, I'm unable to see the participants. Should this still be so?